spec/javascripts/fixtures/PLpeopleLayer.html

```diff
@@ -608,7 +608,7 @@
 
 
         L.layerGroup.pLpeopleLayer = function (options) {
-            return new L.LayerGroup.PLpeopleLayer(options) ;
+            return new L.LayerGroup.pLpeopleLayer(options) ;
```

Comment:
So this P should be in capital. Otherwise this will go in infinite recursion!

Reply:
That causes it to break, there is no function with a capital P?

Reply:
Oh I see now. Two places had to be changed, thanks!